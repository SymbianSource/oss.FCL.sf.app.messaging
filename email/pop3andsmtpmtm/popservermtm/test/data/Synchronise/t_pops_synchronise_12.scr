+OK demon POP3 server ready (demon.co.uk)
+OK username accepted
+OK password accepted
+OK 5 6841
+OK 5 messages
1 1444
2 1359
3 1444
4 1236
5 1358
.
+OK 5 messages
1 t_pops_synchronise:message014
2 t_pops_synchronise:message011
3 t_pops_synchronise:message013
4 t_pops_synchronise:message010
5 t_pops_synchronise:message012
.
-ERR Unknown Command "CAPA"
** Read cancelled
+OK RETR 4 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message010; Wed, 25 Jul 2001 13:41:20 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1129815; 25 Jul 2001 13:40 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010254f6fb2323@teleca161.teleca.com> for <freddy@yashar.demon.co.uk>;
 Wed, 25 Jul 2001 14:55:06 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
	id <PRD792MH>; Wed, 25 Jul 2001 14:43:51 +0100
Message-ID: <t_pops_synchronise_message010@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 10
Date: Wed, 25 Jul 2001 14:37:16 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
	charset="iso-8859-1"

Ten Ten Ten Ten Ten Ten Ten Ten Ten Ten 
Ten Ten Ten Ten Ten Ten Ten Ten Ten Ten 
Ten Ten Ten Ten Ten Ten Ten Ten Ten Ten 
Ten Ten Ten Ten Ten Ten Ten Ten Ten Ten 
.
+OK RETR 3 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message011; Wed, 25 Jul 2001 13:47:28 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1100689; 25 Jul 2001 13:40 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010254f6fb7346@teleca161.
teleca.com> for <freddy@yashar.demon.co.uk>;
 Wed, 25 Jul 2001 14:55:27 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <PRD792M2>; Wed, 25 Jul 2001 14:43:55 +0100
Message-ID: <t_pops_synchronise_message011@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 11
Date: Wed, 25 Jul 2001 14:37:19 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven 
Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven 
Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven 
Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven Eleven 
.
+OK RETR 5 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message012; Wed, 25 Jul 2001 13:49:06 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1101619; 25 Jul 2001 13:40 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac10010254f6fbc3a9@teleca161.
teleca.com> for <freddy@yashar.demon.co.uk>;
 Wed, 25 Jul 2001 14:55:47 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <PRD792MJ>; Wed, 25 Jul 2001 14:44:01 +0100
Message-ID: <t_pops_synchronise_message012@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 12
Date: Wed, 25 Jul 2001 14:37:25 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve 
Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve 
Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve 
Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve Twelve 
.
** Read cancelled
+OK disconnecting


