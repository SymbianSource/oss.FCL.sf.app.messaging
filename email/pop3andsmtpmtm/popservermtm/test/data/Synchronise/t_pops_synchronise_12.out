STAT
LIST
UIDL
CAPA
RETR 4
RETR 2
RETR 5
QUIT
